docs/en/14-reference/02-tools/10-taosbenchmark.md

```diff
@@ -431,11 +435,9 @@ Specify the configuration parameters for tag and data columns in `super_tables`
 
 - **create_table_thread_count** : The number of threads for creating tables, default is 8.
 
-- **connection_pool_size** : The number of pre-established connections with the TDengine server. If not configured, it defaults to the specified number of threads.
-
 - **result_file** : The path to the result output file, default is ./output.txt.
 
-- **confirm_parameter_prompt** : A toggle parameter that requires user confirmation after a prompt to continue. The default value is false.
+- **confirm_parameter_prompt** : A toggle parameter that requires user confirmation after a prompt to continue. The value can be "yes" or "no", the default value is "no".
```

Comment:
The value can be "yes" or "no", by default "no".

Reply:
done